src/Commands/SyncUnionCloudDataUsers.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace BristolSU\UnionCloud\Commands;
+
+use BristolSU\UnionCloud\Jobs\getUsersData;
+use BristolSU\UnionCloud\Jobs\notifyAdmin;
+use BristolSU\UnionCloud\Jobs\processUserData;
+use GuzzleHttp\Exception\ClientException;
+use Illuminate\Console\Command;
+use BristolSU\UnionCloud\UnionCloud\UnionCloud;
+use Illuminate\Support\Facades\Log;
+
+
+class SyncUnionCloudDataUsers extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'unioncloud:users:sync:all';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sync users from UnionCloud';
+
+    // Not currently working as Page Limit doesn't work on the Package.
```

Comment:
Is this something I need to be aware of?

Reply:
We've discussed it before, UC will return the full 500 records as this isn't working on the UC package. I think there is a comment about it on that package. It actually shouldn't be an issue as it just gets the max number of records in 1 go, which seems to be fine.

Reply:
Ah of course, thanks!